data/presets/amenity/fast_food/cafeteria.json

```
@@ -0,0 +1,18 @@
+{
+    "icon": "maki-restaurant",
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "terms": [
+        "eat",
+        "breakfast",
+        "dining",
+        "dinner"
```

I would replace these generic terms with some that are specific to the preset.

```suggestion
        "canteen",
        "dining hall",
        "lunchroom",
        "refectory"
```

---

Yes, I agree